tests/ci/docker_images/linux-x86/ubuntu-20.04_android/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu:20.04
+
+SHELL ["/bin/bash", "-c"]
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+
+ENV ANDROID_SDK_URL commandlinetools-linux-8092744_latest
+ENV ANDROID_HOME /opt/sdk
+
+# ------------------------------------------------------
+# --- Android SDK
+
+RUN set -ex && \
+    apt-get update -y && \
+    apt-get -y --no-install-recommends upgrade && \
+    apt-get -y --no-install-recommends install \
+    python3.8 \
+    python3.8-venv \
+    python3-pip \
+    openjdk-8-jdk \
+    perl \
+    libunwind-dev \
+    wget \
+    unzip && \
+# install go dependencies
+    cd /tmp && \
+    wget https://dl.google.com/go/go1.13.12.linux-amd64.tar.gz && \
+    tar -xvf go1.13.12.linux-amd64.tar.gz && \
+    mv go /usr/local && \
+# install android-sdk from url source
+    mkdir /opt/sdk && \
+    mkdir /opt/sdk/cmdline-tools && \
+    mkdir /opt/cmdline-tools-tmp && \
+    cd /opt/cmdline-tools-tmp && \
+    wget -q https://dl.google.com/android/repository/${ANDROID_SDK_URL}.zip && \
+    unzip  ${ANDROID_SDK_URL}.zip && \
+# move to its final location and export path
+    mv ./cmdline-tools ${ANDROID_HOME}/cmdline-tools/latest && \
+    export PATH=$PATH:$ANDROID_HOME/cmdline-tools/latest/bin && \
+    export PATH=$PATH:$ANDROID_HOME/platform-tools && \
```

Comment:
Removed the path exporting, since it doesn't seem necessary now that we `cd` directly into the `bin` folder.
fyi: The path was permanently modified like line 57, when I did that.